spark-operator-api/src/test/java/org/apache/spark/k8s/operator/status/ApplicationStatusTest.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.spark.k8s.operator.status;
+
+import static org.apache.spark.k8s.operator.status.ApplicationStateSummary.SUBMITTED;
+import static org.apache.spark.k8s.operator.status.ApplicationStateSummary.SUCCEEDED;
```

Comment:
Shall we use `*`?

Reply:
wild card was disabled per [style guide](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports) - but we can fully qualify static imports.

Reply:
Shy.. Too many lines.

Reply:
Ack, if that's the way.